hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -432,7 +443,14 @@ private RaftProperties newRaftProperties(ConfigurationSource conf) {
     final int logPurgeGap = conf.getInt(
         OMConfigKeys.OZONE_OM_RATIS_LOG_PURGE_GAP,
         OMConfigKeys.OZONE_OM_RATIS_LOG_PURGE_GAP_DEFAULT);
-    RaftServerConfigKeys.Log.setPurgeGap(properties, logPurgeGap);
+
+    if (forUpgrade) {
+      LOG.info("Setting purge gap to 1.");
+      RaftServerConfigKeys.Log.setPurgeGap(properties, 1);
+    } else {
+      LOG.info("Setting purge gap to 1000.");
```

Comment:
Nit: purge gap should not hard-coded as 1000 in this log, can you use variable logPurgeGap instead of?

Reply:
Thanks, used it while testing it manually :) Forgot to remove it.